google-cloud-speech/lib/google/cloud/speech/result.rb

```diff
@@ -114,10 +123,40 @@ class Alternative
           ##
           # @private Creates a new Result::Alternative instance.
           def initialize transcript, confidence
-            @transcript  = transcript
+            @transcript = transcript
             @confidence = confidence
           end
         end
+
+        ##
+        # Word-specific information for recognized words. Word information is
+        # only included in the response when certain request parameters are
```

Comment:
That wording comes from the protobuf. It doesn't provide any other context and I'm not aware what other parameters it could mean. I removed that phrase in other documentation entries, but it slipped through here. I'll remove it.